### PR TITLE
[MVPN-334] Move identity registration to vpn page

### PR DESCRIPTION
**Pull request by ignasbernotas**
FInished Vaidotas PR: https://github.com/mysteriumnetwork/mysterium-vpn/pull/366

### CODE REVIEW

File: src/renderer/app.vue [hunk 1]
```diff
@@ -20,8 +20,6 @@
     id="app"
     class="app">
     <div id="content">
-      <IdentityRegistration v-if="paymentsAreEnabled"/>
```
Comment (donce) [inline src/renderer/app.vue]: !! `paymentsAreEnabled` are not used anymore, it should be removed.

File: src/renderer/pages/vpn.vue [hunk 2]
```diff
@@ -21,6 +21,9 @@
       v-if="paymentsAreEnabled && !isIdentityMenuOpen"
       :registered="registered"
       :click="showInstructions"/>
+
+    <IdentityRegistration />
```
Comment (donce) [inline src/renderer/pages/vpn.vue]: ?? Should we check for payments feature toggle to avoid rendering it?

Comment (shroomist) [inline src/renderer/pages/vpn.vue]:
```suggestion
    <IdentityRegistration v-if="paymentsAreEnabled" />
```

File: src/renderer/app.vue [hunk 3]
```diff
@@ -60,15 +57,11 @@ export default {
...
     AppError,
-    AppModal,
-    IdentityRegistration
+    AppModal
   },
   dependencies: ['rendererCommunication', 'syncCommunication', 'logger', 'bugReporterMetrics', 'featureToggle'],
   computed: {
-    ...mapGetters(['navVisible', 'loading', 'visual', 'overlayError']),
-    paymentsAreEnabled () {
-      return this.featureToggle.paymentsAreEnabled()
```
Comment (donce) [inline src/renderer/app.vue]: !! `featureToggle` dependency is not being used anymore, please remove it.

Comment (ignasbernotas) [reply src/renderer/app.vue]: @donce fixed

**Pull request merged**